build.sbt

```diff
@@ -3,6 +3,8 @@ import sbtcrossproject.CrossPlugin.autoImport.crossProject
 
 val commonSettings = Seq(
   organization := "com.github.cb372",
+  scalaVersion := "2.12.8",
```

Comment:
These 2 lines can be deleted from here, because they are picked up from .travis.yml. But don't worry about it, I'll merge this and then fix it on master.